packages/tailwindcss/src/design-system.ts

```diff
@@ -29,7 +29,7 @@ export type DesignSystem = {
   compileAstNodes(candidate: Candidate): ReturnType<typeof compileAstNodes>
 
   getVariantOrder(): Map<Variant, number>
-  resolveThemeValue(path: string): string | undefined
+  resolveThemeValue(path: string, forceInline?: boolean): string | undefined
```

Comment:
I have to make this optional because of IntelliSense unfortunately. Also decided to make a boolean here since don't leak the `ThemeOption` enum beyond internal APIs right now.